src/KDL_server.cpp

```diff
@@ -0,0 +1,119 @@
+#include <kdl_parser/kdl_parser.hpp>
+#include <kdl/chain.hpp>
+#include <kdl/joint.hpp>
+#include <kdl/frames.hpp>
+#include<kdl/chainfksolverpos_recursive.hpp>
+#include <kdl/jntarray.hpp>
+#include <kdl/jacobian.hpp>
+#include <kdl/chainjnttojacsolver.hpp>
+#include <kdl/jntarrayvel.hpp>
+#include <ros/console.h>
+#include <kdl/tree.hpp>
+
+#include "ros/ros.h"
+#include "ros_robotic_skin/getJacobian.h"
+
+
+
+// https://medium.com/@sarvagya.vaish/forward-kinematics-using-orocos-kdl-da7035f9c8e
+// http://mirror.umd.edu/roswiki/pr2_mechanism(2f)Tutorials(2f)Coding(20)a(20)realtime(20)Cartesian(20)controller(20)with(20)KDL.html
+
+class TreeHelper{
```

Comment:
We should change the name of this class to better express what it does. I know I set this, but I don't have a complete understanding here.